ec2uploadimg

```diff
@@ -955,10 +956,17 @@ def upload_image_to_region(
                 logger.info('Created image: {}'.format(str(ami)))
     except EC2UploadImgException as e:
         print_ex(logger, e)
+        errors += 1
     except Exception as e:
         print_ex(logger, e)
+        errors += 1
     finally:
-        setup.clean_up()
+        # avoid unclean termination in case of issues in the cleanup routine
+        try:
+            setup.clean_up()
```

Comment:
If an exception occurs during clean up the code will not reach `sys.exit(1)`, i.e. we are missing part of the intended fix.